packages/blade/src/components/Input/TextArea/TextArea.tsx

```diff
@@ -244,6 +246,7 @@ const _TextArea: React.ForwardRefRenderFunction<BladeElementRef, TextAreaProps>
       }}
       onKeyDown={(e) => {
         handleTaggedInputKeydown(e);
+        onKeyDown?.(e);
```

Comment:
What's e here? {name, value, event}???

Reply:
> {name, value, event}???

I'm fixing it. I just realised its `{name, key, event}`. We can make it `{ name, value, event }`